### PR TITLE
Added boards constants for LED_BUILTIN and KEY_BUILTIN

### PR DESCRIPTION
**Pull request by arcao**
**Sources:**
* esp320: http://wiki.sweetpeas.se/index.php?title=ESP320#Developing_using_the_Arduino_IDE
* esp32thing: https://cdn.sparkfun.com/assets/learn_tutorials/5/0/7/esp32-thing-schematic.pdf
* espea32: https://github.com/AprilBrother/ESPea32/blob/master/hardware/ESPea32-schematic.pdf
* hornbill32dev: https://github.com/ExploreEmbedded/Hornbill-ESP32-DEV/blob/master/Design%20Files/Schematic/Hornbill%20Devboard.pdf
* lolin32: values from existing header
* nano32: http://iot-bits.com/closer-look-esp32-nano32/
* node32s: https://github.com/lamloei/node32s-book/blob/master/ch01/node32s-schematic.pdf

_Note: Some board producers have not published schematics yet, so it was little digging work._

### CODE REVIEW

File: variants/esp320/pins_arduino.h [hunk 1]
```diff
@@ -9,6 +9,12 @@
 #define digitalPinToInterrupt(p)    (((p)<40)?(p):-1)
 #define digitalPinHasPWM(p)         (p < 34)
 
+static const uint8_t LED_BUILTIN = 5;
+static const uint8_t BUILTIN_LED = 5;
```
Comment (me-no-dev) [inline variants/esp320/pins_arduino.h]: since those are the same, wouldn't it be better if one of them is define to forward to the other one?

Comment (arcao) [inline variants/esp320/pins_arduino.h]: Yes, you are right, I will do it. I tried to do it same way as is in ESP8266 core.

**Comment by arcao** [comment]
Done.

**Comment by me-no-dev** [comment]
thanks :)

**Pull request merged**